docs/stripe.md

```diff
@@ -0,0 +1,145 @@
+# Data striping
+
```

Comment:
I'm not sure where this page should go - I don't think it should be its own top level page. Maybe it should be a subsection in here: http://aquaqanalytics.github.io/TorQ/utilities/#pubsubq  ?

Reply:
I have moved the documentation to the subsection of pubsub.q in 70c9c8b833392c818eba63956cc1a024196d4226. However, I am not sure how the website renders the md text. Based on the existing md - it seems like the subsection starts with ### Heading level 3.

Reply:
I have added 7a9279a9f7f78f24730fb10297176dfe95ad2d64 for your reference which uses the same format as the existing md (starting with ### Heading level 3). In terms of visuals on the website, I am not sure if the subheaders for the subsection will be too small for reading.

Reply:
You can use `mkdocs build` & `mkdocs serve` to build & serve the docs and view how they will be rendered, see here: https://github.com/AquaQAnalytics/TorQ#updating-the-documentation-with-mkdocs

Reply:
Great thanks, I have fixed the formatting and added a new paragraph on how general subscribers can make use of the hash function to stripe data.